pyfar/classes/audio.py

```diff
@@ -1,9 +1,12 @@
 """
 The following documents the audio classes and arithmethic operations for
-audio data. More details and background is given in the concepts (
-:py:mod:`audio classes <pyfar._concepts.audio_classes>`,
-:py:mod:`Fourier transform <pyfar._concepts.fft>`,
-:py:mod:`arithmetic operations <pyfar._concepts.arithmetic_operations>`).
+audio data. More details and background is given in the gallery (
```

Comment:
Maybe just a link to the gallery is sufficient?

Reply:
Since there are already lots of examples and more will come - so I think explicitly referencing whats important might help new users.